### DISCUSSION TITLE
%HOME%\Appdata\Roaming\DBeaverData folder does not exist

### DISCUSSION THREAD

**Discussion by FireFighter1017** [dbeaver/dbeaver#23077] [Q&A] [answered]
### Description

I recently installed DBeaver CE version 23.2.2.202401211417
The default project 'General' is using a folder that do not exist for Windows File Explorer.
![image](https://github.com/dbeaver/dbeaver/assets/23411416/d3b7da6d-d684-4c8f-93c3-cd8d15c3cab1)

When trying to locate the folder myself, I found out the 'DBeaverData' folder does not exist, even if I enable showing hidden and system files in windows explorer.
![image](https://github.com/dbeaver/dbeaver/assets/23411416/a39d9ae0-7205-4155-b51d-adcaec4d455c)

### DBeaver Version

23.2.2.202101211417

### Operating System

Windows 11 Enterprise 22H2 build 22621.3155

### Steps to reproduce

Fresh install DBeaver CE

**Comment by Destrolaric** [comment]
@FireFighter1017 Hi, do you use Windows store version?

**Comment by jonasnordlund** [comment]
FWIW, I have the same issue but yeah, I just realized that I do use the store version. I noticed now that I was trying to copy some SQL scripts out of the folder.

**Comment by Destrolaric** [comment]
If Windows store version used the Appdata\Roaming is unfortunately virtualized by Windows, you will need to open the Windows store app folder to get the data.

**Comment by jonasnordlund** [comment, marked as answer]
I just realized that a workaround that might work for you is going here instead:

%LOCALAPPDATA%\Packages

Then find the app starting with DBeaverCorp.DBeaverCE and navigate further via LocalCache\Roaming\DBeaverData. Try to be careful though as this might mess with the app state. I'd only really copy stuff out of there if you need anything. In my case, I needed to get to some SQL scripts that I had saved.

**Comment by Destrolaric** [comment]
@jonasnordlund Thanks for the additional clarification, as soon as @FireFighter1017 confirm that Windows Store version is used, I will convert the issue to discussion with your  message marked as answer.

**Comment by FireFighter1017** [comment]
> @FireFighter1017 Hi, do you use Windows store version?

I don’t think so as it is deployed through our Company Portal.

**Comment by FireFighter1017** [comment]
> I just realized that a workaround that might work for you is going here instead:
> 
> %LOCALAPPDATA%\Packages
> 
> Then find the app starting with DBeaverCorp.DBeaverCE and navigate further via LocalCache\Roaming\DBeaverData. Try to be careful though as this might mess with the app state. I'd only really copy stuff out of there if you need anything. In my case, I needed to get to some SQL scripts that I had saved.

I will try that thanks

**Comment by FireFighter1017** [comment]
> > I just realized that a workaround that might work for you is going here instead:
> > %LOCALAPPDATA%\Packages
> > Then find the app starting with DBeaverCorp.DBeaverCE and navigate further via LocalCache\Roaming\DBeaverData. Try to be careful though as this might mess with the app state. I'd only really copy stuff out of there if you need anything. In my case, I needed to get to some SQL scripts that I had saved.
> 
> I will try that thanks

I found it using your workaround @jonasnordlund 
C:\Users\xxxxx\AppData\Local\Packages\DBeaverCorp.DBeaverCE_xxxxxxx\LocalCache\Roaming\DBeaverData\workspace6

Thanks a lot!

**Comment by FireFighter1017** [comment]
Though DBeaver should be able to figure this one out instead of defaulting to %HOME%\AppData and forcing users to navigate obscur folders to find it.
